.rubocop.yml

```diff
@@ -323,3 +323,18 @@ Rails/I18nLocaleAssignment: # (new in 2.11)
   Enabled: true
 Rails/UnusedIgnoredColumns: # (new in 2.11)
   Enabled: true
+
+Layout/LineEndStringConcatenationIndentation: # new in 1.18
+  Enabled: true
+Lint/AmbiguousOperatorPrecedence: # new in 1.21
+  Enabled: true
+Lint/AmbiguousRange: # new in 1.19
+  Enabled: true
+Lint/IncompatibleIoSelectWithFiberScheduler: # new in 1.21
+  Enabled: true
+Style/RedundantSelfAssignmentBranch: # new in 1.19
+  Enabled: true
+Rails/EagerEvaluationLogMessage: # new in 2.11
+  Enabled: false
```

Comment:
`false` so that we don't need to change our tests around logging behavior. 🤷🏻‍♂️

Reply:
yeah, i can see the point of this cop, but at present, i don't think we'd gain a ton in this code base by enabling it.